test/e2e/events_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+)
+
+// The actual test suite
+var _ = t.Describe("events", func() {
+	It("should succeed", func() {
+		// Given
+		endpoint, testDir, crio := t.StartCrio()
+
+		// When
+		session := t.CrictlWithEndpointNoWait(endpoint, "events")
```

Comment:
why no wait? I think the wait is very much intentional to avoid flakes of slow bootstrap

Reply:
I don't have much experience with this testing framework but what I see in [the docs](https://pkg.go.dev/github.com/onsi/gomega@v1.27.10/gexec#Session.Wait) for the `Wait` method:
```
Wait waits until the wrapped command exits. It can be passed an optional timeout. If the command does not exit within the timeout, Wait will trigger a test failure.
```
As the `events` command doesn't terminate we'll probably have the test fail after waiting for 1 minute.

Maybe I can add a short timeout before `Expect(session.Out).ToNot(Say("unknown method GetContainerEvents")) // no errors` to help with slow bootstrap?

Reply:
oh, you are right. I misundertood.